src/features/shields/select/index.ts

```diff
@@ -41,7 +41,8 @@ export const SelectBox = styled<SelectBoxProps, 'select'>('select')`
   cursor: pointer;
 
   > option {
-    color: ${palette.white};
+    /* DO NOT MODIFY: this is for the dropdown only and has effect only on Windows */
```

Comment:
I'm thinking 'Do not modify' is not helpful enough. At some point this will be modified for one reason or another, and it will be useful to see _why_, within the way the code is currently, this needs to be set as 'black' for windows. Perhaps just pointing to the github issue link will be enough to set the context @cezaraugusto ?

Reply:
sounds good, updated